### DISCUSSION TITLE
ISR not wokring ?

### DISCUSSION THREAD

**Discussion by charlesoestergaard** [vercel/next.js#59289] [Help]
### Summary

I use getStaticPaths with getStaticProps. It always worked, but after I updated to version 14.0.3, it doesn't work anymore.

Nextjs with pages/ folder

This is how it works:

1. A user creates a new profile, and their "slug" gets added to the merchant/utils/merchant-slugs/ endpoint.
2. When the page is viewed for the first time, the server should render the page content.
3. Now the page should be built and statically served for the next person to visit. But I get a 500 Server error when I visit the new user's profile.

Works on localhost, Works if we rebuild the build...

```javascript

export async function getStaticPaths() {
  const res = await API.get<ApiSlugsResponse>("merchant/utils/merchant-slugs/");
  console.log(res.data.slugs);
  const paths = res.data.slugs
    .map((slug) => ({ params: { merchant_name: slug } }))
    .flat();
  return { paths, fallback: "blocking" };
}

export async function getStaticProps({
  params,
}: GetStaticPropsContext<{ merchant_name: string }>) {
  const res = await API.get<ApiMerChantResponse>(
    `merchant/get/${params?.merchant_name}`
  );
  console.log(res.data.merchant);
  const merchant = res.data.merchant;
  return {
    props: {
      merchant,
    },
    revalidate: 60,
  };
}
```

**Comment by Amirjab21** [comment]
I'm also having this problem.

Did you manage to fix it?

**Reply by charlesoestergaard** [reply]
It somehow started working again after i redeployed it on vercel. I have no clue why sorry.
